packages/libllvm/build.sh

```diff
@@ -18,6 +18,7 @@ TERMUX_PKG_CONFLICTS="gcc, clang (<< 3.9.1-3)"
 TERMUX_PKG_BREAKS="libclang, libclang-dev, libllvm-dev"
 TERMUX_PKG_REPLACES="gcc, libclang, libclang-dev, libllvm-dev"
 TERMUX_PKG_GROUPS="base-devel"
+TERMUX_SUBPKG_GROUPS="base-devel"
```

Comment:
subpkg groups can be set only in `*.subpackage.sh`.

Reply:
ok